ga4gh/datarepo.py

```diff
@@ -234,6 +237,14 @@ def printSummary(self):
                     featureSet.getOntology().getName(),
                     featureSet.getId(),
                     sep="\t")
+            print("\tPhenotypeAssociationSets:")
+            for phenotypeAssociationSet in \
+                    dataset.getPhenotypeAssociationSets():
+                print(
+                    "\t", phenotypeAssociationSet.getLocalId(),
+                    phenotypeAssociationSet.getParentContainer().getId(),
+                    sep="\t")
+                # TODO - gabrielle please improve this listing
```

Comment:
we can keep the comment, but just remove the reference to the person that should do it